trace-gradle-plugin/src/functionalTest/java/io/bitrise/trace/plugin/TraceGradlePluginFunctionalTest.java

```diff
@@ -395,9 +338,11 @@ private void verifyReleaseGenerateBuildIdTasks(@NonNull final BuildResult buildR
      */
     private void verifyGenerateBuildIdTasksForBuild() {
         final File debugBuildIdFile =
-                new File(getTestDir().getPath() + "/build/outputs/apk/debug/bitriseBuildId.txt");
+                new File(functionalTestHelper.getTestDir(testName)
+                                             .getPath() + "/build/outputs/apk/debug/bitriseBuildId.txt");
```

Comment:
nit: should these (L342, L345, L358) be constants ? like the other file names in this class?

Reply:
Created a template for the common.